recipes/fftw/build.sh

```diff
@@ -0,0 +1,9 @@
+# Install single precision
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX --enable-shared --disable-static --enable-single
+make
+make install
+# Install double precision
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX --enable-shared --disable-static
+make
+make install
```

Comment:
I don't know `fftw`, but does that create files with different names or does the second build clobber any files from the first?